index.html

```diff
@@ -0,0 +1,218 @@
+              />
+              <p class="program-name">Lecture</p>
+            </div>
+            <p class="program-description">
+             This second edition aims at joining
+             both academy and industry by covering not only basic
+             and applied research,
+             but also philosophical and ethical issues
+             regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img
+                src="./assets/images/exhibition-icon.png"
+                alt="exhibition"
+                class="icon"
+              />
+              <p class="program-name">CC Exhibition</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+             both academy and industry 
+             by covering not only basic and applied research,
+             but also philosophical and ethical issues
+              regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img src="./assets/images/forum-icon.png" alt="forum" class="icon" />
+              <p class="program-name">Forum</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+             both academy and industry by covering
+              not only basic and applied research,
+             but also philosophical and ethical 
+             issues regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img
+                src="./assets/images/workshop-icon.png"
+                alt="workshop"
+                class="icon"
+              />
+              <p class="program-name">Workshop</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+             both academy and industry by covering
+              not only basic and applied research,
+             but also philosophical and ethical
+              issues regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img src="./assets/images/ignite-icon.png" alt="ignite" class="icon" />
+              <p class="program-name">CC Ignite</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+              both academy and industry by covering
+               not only basic and applied research,
+              but also philosophical and ethical
+               issues regarding the future of humanity in an AI world.
+            </p>
+          </li>
+        </ul>
+        <div class="whole-program"><a href="#">SEE THE WHOLE PROGRAM</a></div>
```

Comment:
Please use the same color for all the sections of the home page from the design
![see](https://user-images.githubusercontent.com/35504149/137373155-a4a99bda-b8c2-488a-8707-679999223805.png)
.